content/english/csharp-guess-the-word/activity-1.md

```diff
@@ -7,8 +7,8 @@ weight: 4
 
 ## Instructions
 
-Go to the code containing the [GuessTheWord](https://aka.ms/nuevoguesstheword) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
+Go to the code containing the [GuessTheWord](https://dotnetfiddle.net/lMl7j4) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
```

Comment:
Could you change the link to be of one that opens in a new page?

```
<a href="https://dotnetfiddle.net/lMl7j4" target="_blank">Guess The Word </a>
```

Reply:
clicking on the link does open it in a new window when I try it, so I think we could leave it as is.